pennylane/labs/resource_estimation/ops/op_math/symbolic.py

```diff
@@ -273,17 +1116,82 @@ def tracking_name(
 
 
 class ResourceProd(Prod, re.ResourceOperator):
-    """Resource class for Prod"""
+    r"""Resource class for the symbolic Prod operation.
+
+    A symbolic class used to represent a product of some base operations.
+
+    Args:
+        *factors (tuple[~.operation.Operator]): a tuple of operators which will be multiplied together.
```

Comment:
```suggestion
        * factors (tuple[~.operation.Operator]): a tuple of operators which will be multiplied together.
```

Reply:
The argument is actually *factors. This comes from the docstring of the base `Prod` class